### ISSUE TITLE
Use of uninitialized value $end_marker in quotemeta at /usr/bin/cloc line 4806.

### ISSUE THREAD

**Issue by zilinc** [AlDanial/cloc#166]
I defined a language (Isabelle: https://isabelle.in.tum.de/) as:
```
Isabelle
    filter remove_between_general (* *)
    extension thy
    3rd_gen_scale 3.00

```
in a file X. If I do `cloc * --read-lang-def=X` then it gives lots of warnings: `Use of uninitialized value $end_marker in quotemeta at /usr/bin/cloc line 4806.` (`cloc --version` gives `1.72`). But if I instead write `cloc * --force-lang-def=X` then it works alright.

**Comment by AlDanial** [comment]
Thanks for the bug report.  ``remove_between_general`` was not being parsed correctly with ``--read-lang-def``.  Should be fixed with commit de67b46.

**Comment by zilinc** [comment]
Thanks heaps or the prompt fix.
